tools/EMLassemblyline/make_eml.xml

```diff
@@ -0,0 +1,67 @@
+<tool id="makeeml" name="Make EML" version="0.1.0+galaxy0">
+    <description> Create EML from EAL templates</description>
+    <requirements>
+        <requirement type="package" version="4.3.1">r-base</requirement>
+        <requirement type="package" version="3.5.5">r-emlassemblyline</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    mkdir output_template &&
+    cd output_template &&
+    #for $input in $templates
+        cp '$input' '${input.element_identifier}.txt' &&
+    #end for
+    find . -type f -empty -print -delete &&
+    Rscript '$__tool_directory__/make_eml.R'
+    ]]></command>
+    <inputs>
+        <param name="templates" type="data_collection" type_collection="list" format="tabular" label="Upload all EAL templates" help= "Input a data collection." />
+    </inputs>
+    <outputs>
+        <data name="metadata" format="xml" label="Metadata">
+            <discover_datasets directory = "output_template" pattern= "(?P&lt;designation&gt;.+)\.xml" visible="true"/>
+        </data>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="1"> 
+            <param type="data_collection" name="templates" value="Assessing_the_importance_of_field_margins_for_bat_species.xml">
+                <collection type="list">
+                    <element file="annotations.tabular"/>
+                    <element file="keywords.tabular" />
+                    <element file="attributes_data_blary_al.tabular"/>
+                    <element file="catvars_data_blary_al.tabular"/>
+                    <element file="keywords.tabular" />
+                    <element file="geographic_coverage.tabular" />
+                    <element file="personnel.tabular" />
+                </collection>
+            </param>
+            <output name="metadata" ftype="xml" file="Metadata.xml"/>

```

Comment:
this test expects a single output, but you have defined in the output section a collection as output.

Reply:
Seems ok no?

Reply:
output collection are specified this way?

https://github.com/galaxyecology/tools-ecology/pull/74/files#diff-7e85ffa1db3b0bc3de3c0649530c0e07f5fd1deaa19f24d2dc2a8f48bc702f36R24

Is the output that you expect from this tool a collection or a single dataset?

Reply:
A single dataset. This tool create a metadata file following EML metadata standards specification from existing templates metadata files.

Reply:
Than the output section needs to be adopted